src/routes/Landing.js

```diff
@@ -35,12 +35,17 @@ class Landing extends Component {
             ...params,
             isModalOpen: params && Object.keys(params).length > 0
         });
-
+        
         window.insights.chrome.auth.getUser().then(user => {
             if (user) {
                 this.setState({ unauthed: false });
             } else {
-                this.setState({ unauthed: true });
+                const authResponse = window.insights.chrome.auth.challengeAuth();
```

Comment:
`challengeAuth` is promis so this wouldn't really work. The best way how to branch this would be trough `then and chatch`
```JS
window
.insights
.chrome
.auth
.challengeAuth()
.then((authed) => this.setState({ unauthed: !authed }))
.catch(() => this.setState({ unauthed: true }));
```

Reply:
Fixed!

Reply:
fixed!

Reply:
fixed!

Reply:
fixed!